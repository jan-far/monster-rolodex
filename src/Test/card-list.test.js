import React from "react";
import { shallow } from "enzyme";
import CardList from "../Components/Card-list";

describe("CardList Tests", () => {
  const mockMonster = [{ id: 1, name: "ralph", email: "robo@gmail.com" }];
  it("Make a snapshot of CardList comonent", () => {
    expect(shallow(<CardList monsters={mockMonster} />)).toMatchSnapshot();
  });
});
