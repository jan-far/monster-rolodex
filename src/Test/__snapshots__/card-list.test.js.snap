// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList Tests Make a snapshot of CardList comonent 1`] = `ShallowWrapper {}`;
